app/packages/tiling/src/views/TilingInspectorSidebar/TilingInspectorSidebar.module.css

```
@@ -0,0 +1,18 @@
+.json {
+  margin: 0;
+  padding: 8px 10px;
+  border-radius: 4px;
+  background: var(--color-content-bg-card-2);
+  border: 1px solid var(--color-content-border-subtle);
+  font-family: monospace;
+  font-size: 0.7rem;
+  line-height: 1.4;
+  color: var(--color-content-text-primary);
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)